src/main/java/salesforce/ui/pages/quicktext/NewQuickText.java

```diff
@@ -0,0 +1,137 @@
+package salesforce.ui.pages.quicktext;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+/**
+ * Form for create a New Quick Text.
+ */
+public class NewQuickText extends BasePage {
+
+    private static final String COMBO_BOX = "//span[text() ='%s']//..//..//a[@role='button']";
+    private static final String CHANNEL_XPATH = "//select //option[contains(@value,'%s')]";
+    private static final String BTN_CSS = "button[title = '%s']";
+
+    @FindBy(css = "input[aria-required='true']")
+    private WebElement nameFld;
+
+    @FindBy(id = "quickTextMessageInputTextArea")
+    private WebElement messageFld;
+
+    @FindBy(css = "button[title = 'Save']")
+    private WebElement saveBtn;
+
+    @FindBy(css = "div[class='messageFieldContainer'] .select[aria-required='false']")
+    private WebElement fieldComboBox;
```

Comment:
In some applications the fields in forms are configurable and the user can decide which ones are required and which ones are not.